task-sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -428,6 +428,9 @@ class WatchedSubprocess:
     subprocess_logs_to_stdout: bool = False
     """Duplicate log messages to stdout, or only send them to ``self.process_log``."""
 
+    start_time: float | None = None
```

Comment:
```suggestion
    start_time: float = attrs.field(factory=time.monotonic)
```
Is probably the best way to handle this actually